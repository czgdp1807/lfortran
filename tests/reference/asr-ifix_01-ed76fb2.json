{
    "basename": "asr-ifix_01-ed76fb2",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/ifix_01.f90",
    "infile_hash": "071fd2fff88805039bb2f2d28c599704b137261b241efff1c3b430c4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-ifix_01-ed76fb2.stderr",
    "stderr_hash": "93f615368596f442b7ab91499cfd3f3863ea79aae6ede7aaf3761d2c",
    "returncode": 2
}