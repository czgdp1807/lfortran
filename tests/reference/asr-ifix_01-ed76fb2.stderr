semantic error: first argument of `ifix` must have kind equals to 4
 --> tests/errors/ifix_01.f90:2:14
  |
2 |     print *, ifix(4.23_8)
  |              ^^^^^^^^^^^^ 
